roles/tas_single_node/defaults/main.yml

```diff
@@ -62,89 +60,29 @@ tas_single_node_tsa_certificate_chain_filename: tsa-certificate-chain.pem
 tas_single_node_tsa_leaf_certificate_filename: tsa-leaf-certificate.pem
 tas_single_node_tsa_signer_private_key_filename: tsa-signer-private-key.pem
 
-tas_single_node_remote_private_key: "{{ tas_single_node_certs_dir }}/{{ tas_single_node_private_key_filename }}"
-tas_single_node_remote_ca: "{{ tas_single_node_certs_dir }}/{{ tas_single_node_ca_filename }}"
-tas_single_node_remote_fulcio_private_key: "{{ tas_single_node_certs_dir }}/{{ tas_single_node_fulcio_private_key_filename }}"
-tas_single_node_remote_fulcio_public_key: "{{ tas_single_node_certs_dir }}/{{ tas_single_node_fulcio_public_key_filename }}"
-tas_single_node_remote_fulcio_root_ca: "{{ tas_single_node_certs_dir }}/{{ tas_single_node_fulcio_root_ca_filename }}"
-tas_single_node_remote_ctlog_private_key: "{{ tas_single_node_certs_dir }}/{{ tas_single_node_ctlog_private_key_filename }}"
-tas_single_node_remote_ctlog_public_key: "{{ tas_single_node_certs_dir }}/{{ tas_single_node_ctlog_public_key_filename }}"
-tas_single_node_remote_rekor_signer: "{{ tas_single_node_certs_dir }}/{{ tas_single_node_rekor_signer_filename }}"
-tas_single_node_remote_rekor_public_key: "{{ tas_single_node_certs_dir }}/{{ tas_single_node_rekor_public_key_filename }}"
-
-tas_single_node_rekor_enabled: true
 tas_single_node_rekor_public_key_retries: 5
 tas_single_node_rekor_public_key_delay: 10
-tas_single_node_fulcio_server_config: "{{ tas_single_node_kube_configmap_dir }}/fulcio-config.yaml"
-tas_single_node_fulcio_server_secret_config: "{{ tas_single_node_kube_configmap_dir }}/fulcio-server-secret.yaml"
-tas_single_node_ctlog_secret: "{{ tas_single_node_kube_configmap_dir }}/ctlog-secret.yaml"
-tas_single_node_ctlog_configmap_config: "{{ tas_single_node_kube_configmap_dir }}/ctlog-config.yaml"
-tas_single_node_tuf_secret_config: "{{ tas_single_node_kube_configmap_dir }}/tuf-secret.yaml"
-tas_single_node_treeid_config: "{{ tas_single_node_kube_configmap_dir }}/treeid-config.yaml"
-tas_single_node_rekor_sharding_config: "{{ tas_single_node_kube_configmap_dir }}/rekor-sharding-config.yaml"
-tas_single_node_nginx_config: "{{ tas_single_node_kube_configmap_dir }}/nginx-config.yaml"
-tas_single_node_nginx_certs_config: "{{ tas_single_node_kube_configmap_dir }}/nginx-certs.yaml"
-tas_single_node_trillian_secret: "{{ tas_single_node_kube_configmap_dir }}/trillian-secret.yaml"
-tas_single_node_systemd_directory: /etc/systemd/system
 
 tas_single_node_setup_host_dns: true
 tas_single_node_base_hostname: ""
 
-tas_single_node_fulcio_enabled: true
+
 tas_single_node_fulcio_ca_passphrase: rhtas
```

Comment:
I would say let's move these to vars as well, same reason as for the certs/keys themselves.